py/CHANGELOG.md

```diff
@@ -100,18 +92,17 @@ This release requires Python 3.8 or later.
 ## 0.8.27
 
 - Add is_enabled flag on transaction filter. ([#2251](https://github.com/getsentry/relay/pull/2251))
+- Add trace context to CheckIns. ([#2241](https://github.com/getsentry/relay/pull/2241))
 
 ## 0.8.26
 
 - Add filter based on transaction names. ([#2118](https://github.com/getsentry/relay/pull/2118))
 - Add `lock` attribute to the frame protocol. ([#2171](https://github.com/getsentry/relay/pull/2171))
-- Add trace context to CheckIns. ([#2241](https://github.com/getsentry/relay/pull/2241))
+
 
 ## 0.8.25
 
-### Various fixes & improvements
 
-- release: 0.8.24 (a9a78dfa) by @getsentry-bot
```

Comment:
I did not see an obvious change between these releases that I could write down, and I did not want to spend more time investigating.